frontend/src/views/mspcontracts/GetGroupMembersContractAddress.vue

```diff
@@ -0,0 +1,208 @@
+<template>
+  <div>
+    <form @submit.prevent="submitForm">
+      <AppRow>
+        <AppCol class="col3">
+          <AppInput :e-model="v$.groupNumber" id="groupNumber" label="Group Number" type="text" v-model.trim="groupNumber" />
+        </AppCol>
+      </AppRow>
+      <AppRow>
+        <AppCol class="col3">
+          <AppInput :e-model="v$.phn" id="phn" label="PHN" type="text" v-model.trim="phn" />
+        </AppCol>
+      </AppRow>
+      <AppRow>
+        <AppButton :submitting="searching" mode="primary" type="submit">Submit</AppButton>
+        <AppButton @click="resetForm" mode="secondary" type="button">Clear</AppButton>
+      </AppRow>
+    </form>
+  </div>
+  <br />
+  <div v-if="searchOk && result.contractInquiryBeneficiaries.length > 0">
+    <hr />
+    <AppSimpleTable id="resultsTable">
+      <thead>
+        <tr>
+          <th>PHN</th>
+          <th>Surname</th>
+          <th>First Name</th>
+          <th>Second Name</th>
+          <th>Third Name</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <!-- Only the first row should be shown as that will be the searched on Group Member -->
+          <GroupMemberContractBeneficiary :beneficiary="result.contractInquiryBeneficiaries[0]" />
```

Comment:
I think it would be cleaner to store the single row as data rather than the whole result. Nevermind, I see that other parts of the result are used. How about a computed field then?